tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -680,11 +732,11 @@ ZTEST_USER(uart_async_chain_write, test_chained_write)
 		      "RX_DISABLED timeout");
 }
 
-ZTEST_BMEM uint8_t long_rx_buf[1024];
-ZTEST_BMEM uint8_t long_rx_buf2[1024];
-ZTEST_BMEM uint8_t long_tx_buf[1000];
-ZTEST_BMEM volatile uint8_t evt_num;
-ZTEST_BMEM size_t long_received[2];
```

Comment:
I don't think you should change the array sizes. This is intentional and is used to test the driver's implementation. For example, it currently checks that the right number of UART_RX_RDY calls are made with this set up of arrays in the test_long_buffers test.

Reply:
Fixed